plugins/tiddlywiki/browser-storage/startup.js

```diff
@@ -53,6 +54,48 @@ exports.startup = function() {
 		$tw.wiki.addTiddler({title: ENABLED_TITLE, text: "no"});
 		$tw.browserStorage.clearLocalStorage();
 	});
+	// Helpers for protecting storage from eviction
+	var setPersistedState = function(state) {
+			$tw.wiki.addTiddler({title: PERSISTED_STATE_TITLE, text: state});
+		},
+		requestPersistence = function() {
+			setPersistedState("requested");
+			navigator.storage.persist().then(function(persisted) {
+				console.log("Request for persisted storage " + (persisted ? "granted" : "denied"));
```

Comment:
IMO logging is a bit excessive. TW has a logger utility function. may be this one can be used. .. But I do not really know how to use it. https://github.com/Jermolene/TiddlyWiki5/blob/master/core/modules/utils/logger.js 

@Jermolene Is there some docs, how it should be used?

Reply:
Are you suggesting I remove the log statement? The state is being captured in a tiddler, so it is redundant and maybe not needed?

I see elsewhere in this file, the logger.js you mention is already being instantiated:

```js
	var logger = new $tw.utils.Logger("browser-storage",{
			colour: "cyan"
		});
```
But the only place that logger is being used is when browser storage gets a `QuotaExceededError`. There are many other places in browser storage where console.log is being called directly.

Reply:
Hi @pmario @btheado I think that given that the browser-storage plugin already makes liberal use of `console.log` it's reasonable to continue to use it.

Reply:
I'm OK with liberal use of console.log ... but I want to deactivate is somehow. It's a pain to debug eg: the server version, because every minute it spams the log with new info with no real value.

Reply:
> I'm OK with liberal use of console.log ... but I want to deactivate is somehow. It's a pain to debug eg: the server version, because every minute it spams the log with new info with no real value.

I don't think that that concern applies with the browser storage plugin because it doesn't run under Node.js